src/core_plugins/elasticsearch/lib/expose_client.js

```diff
@@ -55,9 +55,19 @@ module.exports = function (server) {
       ssl.ca = options.ca.map(readFile);
     }
 
+    const host = {
+      host: uri.hostname,
+      port: uri.port,
+      protocol: uri.protocol.replace(':', ''),
```

Comment:
replacing the ":" isn't necessary, and protocol will be null if the parsed uri is just `//localhost` or something... just a thought